### PR TITLE
Adding missing instances of "nextLink"

### PR DESCRIPTION
**Pull request by marstr**
This is in support of
https://github.com/Azure/azure-sdk-for-go/issues/820

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

---

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by mcardosos** [comment]
From the swagger side this is good. But service teams must approve this changes before merging, yes.

**Comment by marstr** [comment]
@tfitzmac and @EvgenyAgafonchikov, I've included you folks because you were the most recent people to touch what looked like relevant Swagger documents (Resources and Networking respectively) in Git History. If you don't feel comfortable signing off on this, I'd appreciate a heads up, or even better a recommendation on a better person. Thanks for your time!

File: specification/network/resource-manager/Microsoft.Network/2015-06-15/usage.json
Comment (marstr) [inline specification/network/resource-manager/Microsoft.Network/2015-06-15/usage.json]: Example of this property being added exactly like this in a later API Version:
https://github.com/Azure/azure-rest-api-specs/blob/7c6c49a6d8507db1b70909b6b380d7f3bd4168cc/specification/network/resource-manager/Microsoft.Network/2017-06-01/usage.json#L140

File: specification/network/resource-manager/Microsoft.Network/2015-05-01-preview/network.json
Comment (marstr) [inline specification/network/resource-manager/Microsoft.Network/2015-05-01-preview/network.json]: Example of this propery being added exactly like this in a later API Version:

https://github.com/Azure/azure-rest-api-specs/blob/7c6c49a6d8507db1b70909b6b380d7f3bd4168cc/specification/network/resource-manager/Microsoft.Network/2017-06-01/usage.json#L140

File: specification/resources/resource-manager/Microsoft.Resources/2015-11-01/resources.json
Comment (marstr) [inline specification/resources/resource-manager/Microsoft.Resources/2015-11-01/resources.json]: Example of this property being added exactly like this in a later API Version:

https://github.com/Azure/azure-rest-api-specs/blob/7c6c49a6d8507db1b70909b6b380d7f3bd4168cc/specification/resources/resource-manager/Microsoft.Resources/2016-07-01/resources.json#L2378

**Review by marstr — COMMENTED** [review]
Giving context, showing how these RPs use the same pattern in later API Versions.

**Review by mcardosos — APPROVED** [review]
Do not merge until service team approves

**Review by EvgenyAgafonchikov — APPROVED** [review]

**Comment by marstr** [comment]
Couldn't get a hold of more service team reviewers, and this is a minor enough change to an old enough Swagger that I'm not worried about it. Further more, it is absolutely required to have functioning code generated in most languages when this API Version is targeted. As more languages adopt Multi-API Version support, each of them will need this change to be merged in.

**Pull request merged**
